mmcv/ops/conv2d_gradfix.py

```diff
@@ -283,15 +283,36 @@ def forward(ctx, grad_output, input):
                     output_padding=output_padding,
                     output_mask=[0, 1, 0])[1]
             else:
-                # General case => cuDNN.
-                name = ('aten::cudnn_convolution_transpose_backward_weight'
-                        if transpose else
-                        'aten::cudnn_convolution_backward_weight')
-                flags = [
-                    torch.backends.cudnn.benchmark,
-                    torch.backends.cudnn.deterministic,
-                    torch.backends.cudnn.allow_tf32
-                ]
+                is_rocm_pytorch = False
+                try:
+                    from torch.utils.cpp_extension import ROCM_HOME
+                    if ((torch.version.hip is not None)
+                            and (ROCM_HOME is not None)):
+                        is_rocm_pytorch = True
+                except ImportError:
+                    pass
+
+                name = ''
+                flags = []
+
+                if is_rocm_pytorch:
```

Comment:
```suggestion
                if is_rocm_pytorch():
```

Reply:
We can reuse this function from mmengine.

```python
from mmengine.utils.dl_utils import is_rocm_pytorch
```

Reply:
There is no is_rocm_pytorch in python3.7